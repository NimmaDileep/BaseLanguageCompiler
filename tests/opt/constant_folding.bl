var x: Int = 8 + 2;
print(x);

var y: Int = 20 - 15;
print(y);

var z: Int = 10 * 2;
print(z);

var p: Int = 50 % 5;
print(p);

var q: Boolean = 10 < 20;
print(q);

var r: Boolean = 1 > 9;
print(r);

var s: Boolean = 3 <= -3;
print(s);

var t: Boolean = 38 >= 28;
print(t);

var u: Boolean = 5 == 5;
print(u);
