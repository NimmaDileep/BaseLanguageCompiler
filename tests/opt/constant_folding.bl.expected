10
5
20
0
true
false
false
true
true